10_construct-database/02_county_construct-databases_race-ethnicity.qmd

```diff
@@ -230,8 +184,9 @@ clean_data <- function(db_unclean){
   db <- db_unclean %>%
     rename_with(~str_replace_all(.x, "learning_rate", "rate_learning")
                 ) %>%
-    rename_with(.cols = starts_with("pctl"), 
-              ~str_replace_all(.x, "pctl_", "pctl_income_"))
+    mutate(
+      pctl_income_quality = pctl_20_quality 
```

Comment:
I'm assuming that the data website is setup such that the income quality variable is not needed for each income percentile shown - in which case this change makes sense.